submissions/xT0PERx/Popup-style/style.css

```diff
@@ -0,0 +1,94 @@
+/*                                         Верхнее меню с 5 блоками  */
+.container {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    font-size: 1em;
+    text-align: center;
+    padding: 5px;
+}
+
+.container div {
+    width: 80px;
+    height: 80px;
+    margin: 5px;
+    cursor: pointer;
+}
+
+img {
+    width: 40px;
+    height: 40px;
+}
+
+label img:hover {
+    border: 1px solid #1d92bd;
+    border-radius: 20%;
+}
+
+.icons div:hover {
+    border: 1px solid #1d92bd;
+}
+
+.main_tullbar input {
+    /* прячу поле для галочки */
+    visibility: hidden;
```

Comment:
Hide checkbox in the right way. Watch this [video](https://www.youtube.com/watch?v=E6kLaaQFctU&t=413s).